proto-public/pbmesh/v1alpha1/proxy_state.proto

```diff
@@ -5,51 +5,51 @@ syntax = "proto3";
 
 package hashicorp.consul.mesh.v1alpha1;
 
-import "pbmesh/v1alpha1/access_logs.proto";
-import "pbmesh/v1alpha1/cluster.proto";
-import "pbmesh/v1alpha1/endpoints.proto";
-import "pbmesh/v1alpha1/escape_hatches.proto";
-import "pbmesh/v1alpha1/listener.proto";
-import "pbmesh/v1alpha1/references.proto";
-import "pbmesh/v1alpha1/route.proto";
-import "pbmesh/v1alpha1/transport_socket.proto";
+import "pbmesh/v1alpha1/pbproxystate/access_logs.proto";
+import "pbmesh/v1alpha1/pbproxystate/cluster.proto";
+import "pbmesh/v1alpha1/pbproxystate/endpoints.proto";
+import "pbmesh/v1alpha1/pbproxystate/escape_hatches.proto";
+import "pbmesh/v1alpha1/pbproxystate/listener.proto";
+import "pbmesh/v1alpha1/pbproxystate/references.proto";
+import "pbmesh/v1alpha1/pbproxystate/route.proto";
+import "pbmesh/v1alpha1/pbproxystate/transport_socket.proto";
 import "pbresource/resource.proto";
 
 message ProxyStateTemplate {
   // proxy_state is the partially filled out ProxyState resource. The Endpoints, LeafCertificates and TrustBundles fields will need filling in after the resource is stored.
   ProxyState proxy_state = 1;
 
   // required_endpoints is a map of arbitrary string names to endpoint refs that need fetching by the proxy state controller.
-  map<string, EndpointRef> required_endpoints = 2;
+  map<string, hashicorp.consul.mesh.v1alpha1.pbproxystate.EndpointRef> required_endpoints = 2;
```

Comment:
Since these are in a subdirectory, I wonder if you can just refer to them as `pbproxystate.EndpointRef` and the compiler will know how to find it?

Reply:
Nice it looks like that works! I'll update